--- /opt/omero/web/OMERO.web/lib/python/omeroweb/settings.py.orig	2017-10-26 13:01:31.615664591 +0000
+++ /opt/omero/web/OMERO.web/lib/python/omeroweb/settings.py	2017-10-26 13:07:04.219434884 +0000
@@ -87,7 +87,7 @@
     ' (proc.%(process)5.5d) %(funcName)s():%(lineno)d'
     ' HTTP %(status_code)d %(request)s')
 
-LOGGING_CLASS = 'omero_ext.cloghandler.ConcurrentRotatingFileHandler'
+LOGGING_CLASS = 'logging.StreamHandler'
 
 LOGGING = {
     'version': 1,
@@ -112,19 +112,11 @@
         'default': {
             'level': 'DEBUG',
             'class': LOGGING_CLASS,
-            'filename': os.path.join(
-                LOGDIR, 'OMEROweb.log').replace('\\', '/'),
-            'maxBytes': 1024*1024*5,  # 5 MB
-            'backupCount': 10,
             'formatter': 'standard',
         },
         'request_handler': {
             'level': 'DEBUG',
             'class': LOGGING_CLASS,
-            'filename': os.path.join(
-                LOGDIR, 'OMEROweb.log').replace('\\', '/'),
-            'maxBytes': 1024*1024*5,  # 5 MB
-            'backupCount': 10,
             'filters': ['require_debug_false'],
             'formatter': 'full_request',
         },
